.gitignore

```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```

Comment:
I asked Fer about it today, and he explained that there is something called "sudoers" that I can look into where I can modify the "group settings" of the pg user. But I feel like it might eat too much into my time on the project.